integration_tests/test_suites/k8s-test-suite/tests/test_external_asset.py

```diff
@@ -164,3 +166,50 @@ def number_y(
     mats = result.asset_materializations_for_node(number_y.op.name)
     assert "is_even" in mats[0].metadata
     assert mats[0].metadata["is_even"].value is True
+
+
+def test_use_excute_k8s_job(namespace, cluster_provider):
+    docker_image = get_test_project_docker_image()
+
+    @asset
+    def number_y_job(context: AssetExecutionContext):
+        core_api = kubernetes.client.CoreV1Api()
+        reader = K8sPodLogsMessageReader()
+        with setup_ext_protocol(
+            context,
+            ExtEnvContextInjector(),
+            reader,
+            extras={"storage_root": "/tmp/"},
+        ) as protocol_env:
+            job_name = "number_y_asset_job"
+
+            # stand-in for any means of creating kubernetes work
+            execute_k8s_job(
```

Comment:
even as an example, woudl love to replace this with a pod variant asap (should not block commit though)

Reply:
Can you expand on your motivation here? My thinking was that this is a fine demonstration of running a k8s pod via some other means than the ext client - with going through a `Job` being a nice proxy for manifesting one through some other Custom Resource Definition

Reply:
Yeah completely fine for now. Feel free to ignore. 

I just think an eventual example that we externalize should launch a pod with some weird bespoke config rather than invoke something like `execute_k8s_job`, which people are liable to blindly copy paste